asv/www/asv.js

```diff
@@ -239,6 +239,7 @@ $(document).ready(function() {
                          "LPT2", "LPT3", "LPT4", "LPT5", "LPT6", "LPT7", "LPT8",
                          "LPT9"];
         name = name.replace(bad_re, "_");
+        name = name.replace("#", "%23");
```

Comment:
Also take the comment above "The implementation must match asv.util.sanitize_filename" into account.

Reply:
This is just escaping the # character. Filename is the same.

Reply:
Or, possibly, this is not the right place for the url quoting, but it should be done at the point where the json request is made.